ambry-store/src/test/java/com/github/ambry/store/StorageManagerTest.java

```diff
@@ -357,15 +369,26 @@ public void replicaFromStandbyToInactiveTest() throws Exception {
       assertEquals("Error code doesn't match", StoreNotStarted, e.getErrorCode());
     }
     storageManager.startBlobStore(localReplica.getPartitionId());
-    // 4. success case (verify both replica's state and decommission file)
+    // 5. store is disabled due to disk I/O error
+    BlobStore localStore = (BlobStore) storageManager.getStore(localReplica.getPartitionId());
+    AtomicBoolean mockDisabledOnError = new AtomicBoolean(true);
+    localStore.setDisabledOnError(mockDisabledOnError);
```

Comment:
why pass in a new object instead the one in localStore?

Reply:
This is to override boolean variable `disabledOnError` for testing purpose. We are not allowed to directly modify the `disabledOnError` within blobstore. Hope this makes sense.